.github/workflows/docs.yml

```diff
@@ -37,7 +37,10 @@ jobs:
           cat configmap-input-vars.yaml.tmpl | envsubst > config-server-repo/artifacts/in/configmap-input-vars.yaml
           cd config-server-repo
           make artifacts
+          cd ..
           # remove the HEADER from artifacts
           find ./ -name *yaml -exec grep -l "####### HEADER END #######" {} ';' | xargs -I{} gawk -i inplace '/####### HEADER END #######/ {p=1;next}p' {}
+          # Replace version numbers in troubleshooting guide with env.
+          cat docs/user-guide/troubleshooting.md | envsubst > docs/user-guide/troubleshooting.md
```

Comment:
shall we better go from md.tmpl to .md. otherwise you run this locallly and then the changed template might be gone ...